src/lib/utils/audio.ts

```diff
@@ -59,7 +60,7 @@ const resolveSpotifyTracks = (data: ResolvedSpotifyData) => {
 
 export type Playlist = { name: string } & (
 	| { type: PlaylistType.Lavalink; tracks: string[] }
-	| ({ type: PlaylistType.Spotify; tracks: ({ name: string; artist: string } & SpotifyAdditions)[] } & SpotifyAdditions)
+	| ({ type: PlaylistType.Spotify; tracks: ({ name: string; artists: string[] } & SpotifyAdditions)[] } & SpotifyAdditions)
```

Comment:
You'd need to do this since it's not in the base object
```suggestion
	| ({ type: PlaylistType.Spotify; tracks: ({ name: string; } & SpotifyAdditions)[] } & Exclude<SpotifyAdditions, 'artists'>)
```